scripts/pylib/pytest-twister-harness/src/twister_harness/__init__.py

```diff
@@ -2,4 +2,10 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+# flake8: noqa
```

Comment:
`flake8` isn't being used AFAIK?

Reply:
Yes, it isn't. We were used it in twisterV2, from where we copied part of the code to `pytest-twister-harness`. So we still try to keep the code clean (maybe we will enable part of flake8 and/or mypy checking to twister)... I added it also to keep my IDE (VS Code) quiet.